src/node/rpc/test/member_voting_test.cpp

```diff
@@ -1478,9 +1491,60 @@ DOCTEST_TEST_CASE("Passing operator vote")
     DOCTEST_CHECK(proposer_vote != votes.end());
     DOCTEST_CHECK(proposer_vote->second == vote_for);
   }
+
+  auto new_operator_kp = tls::make_key_pair();
```

Comment:
Should we also add tests for the negative paths, i.e. operator member trying to add or retire a non-operator member?

Reply:
Good call - I've added a few negative tests as well.